book/src/bridge/extern_rustqt.md

```diff
@@ -30,6 +30,10 @@ A bridge module may contain zero or more `extern "RustQt"` blocks.
 This complements the [`extern "Rust"` CXX section](https://cxx.rs/extern-rust.html)
 but allows for declaring Qt specific features on C++ types.
 
+## Automatic case conversion
+
+Automatically converting to camel or snake case can be done through an attribute for "RustQt" and "C++Qt" blocks, see [here](./attributes.md#automatic-case-conversion)
```

Comment:
Would change this to the following and remove the header, so it's just a sentence in the previous block

```
Automatically converting to camel or snake case can be enabled through an [attribute](./attributes.md#automatic-case-conversion).
```